plugins/bit_invader/bit_invader.cpp

```diff
@@ -194,6 +196,12 @@ void bitInvader::saveSettings( QDomDocument & _doc, QDomElement & _this )
 
 void bitInvader::loadSettings( const QDomElement & _this )
 {
+	// Clear wavetable before loading a new
```

Comment:
From what I can tell, this loop be replaced by `m_graph->model()->clear();`. Either that or `clearInvisible()` just before the LED initialization, at which point the graph length should be updated. The first option seems simpler to me, but all the `xWaveClicked()` functions use `clearInvisible()` so maybe there's a reason to do it that way.

Reply:
Cool, I'll look into it! Thanks!

Reply:
Fixed!